frontend/packages/metal3-plugin/src/components/table-filters.ts

```diff
@@ -0,0 +1,33 @@
+import * as _ from 'lodash';
+
+import { TableFilter } from '@console/internal/components/factory/table-filters';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { getSimpleHostStatus } from '../utils/host-status';
+import { HOST_STATUS_TITLES } from '../constants';
+
+type FilterItem = {
+  id: string;
+  title: string;
+};
+
+type Filter = {
```

Comment:
@vojtechszocs Those types are common across all filters (IIUC), should I move them into `@console/internal/components/factory/table-filters` to make them widely available?

Reply:
Within #1689, I am moving them under `@console/shared`

Reply:
Rebased on that change